cmd/cluster/dynatrace/logsCmd.go

```diff
@@ -144,3 +178,21 @@ func GetQuery(hcpCluster HCPCluster) (query DTQuery, error error) {
 
 	return q, nil
 }
+
+func GetClusterID() (string, error) {
+	cfg, err := clientcmd.BuildConfigFromFlags("", clientcmd.NewDefaultPathOptions().GetDefaultFilename())
+	if err != nil {
+		return "", err
+	}
+	parsedURL, err := url.Parse(cfg.Host)
+	if err != nil {
+		return "", err
+	}
+	re := regexp.MustCompile("/?backplane/cluster/([a-zA-Z0-9]+)/?")
+	matches := re.FindStringSubmatch(parsedURL.Path)
+	if len(matches) < 2 {
+		return "", fmt.Errorf("couldn't find cluster-id from the backplane cluster url")
+	}
+	clusterID := matches[1]
+	return clusterID, nil
+}
```

Comment:
This is a re-implementation of https://github.com/openshift/osdctl/blob/master/pkg/k8s/client.go#L166
I think we can use the existing function instead of adding a new one which does the same